packages/circuit/tests/circuit/test_parameter_mapping.py

```diff
@@ -52,7 +52,7 @@ def m(vals: ParameterValueAssignment) -> float:
 
                 return m
 
-            def is_trivial_mapping(self) -> bool:
+            def is_trivial_mapping(self) -> bool:  # type: ignore[override]
```

Comment:
This will fix the error
```suggestion
            @property
            def is_trivial_mapping(self) -> bool:
```